pheweb/serve/data_access/db.py

```diff
@@ -780,89 +778,96 @@ def get_variant_results_range(self, chrom, start, end):
                 "No variants in the given range. {}:{}-{}".format(chrom, start - 1, end)
             )
             return []
+        
+        maf_col = (
```

Comment:
Can we put the names of the columns (af cols, #pheno, mlogp...) to a config file and read them from there? Makes the logic clearer and then we don't have to change the code if the columns are different

Reply:
Not fully sure if I understood correctly... Could you please take a look on the last commit and let me know if this is what you had in mind? 

Now the columns can be specified in the config.py file as 

### TabixResultDao:
 "TabixResultDao": { 'const_arguments': [...], "columns": {"pval": "pval", "mlogp": "mlogp", "beta": "beta", "sebeta": "sebeta", "maf": "af_alt", "maf_cases": "af_alt_cases", "maf_controls": "af_alt_controls"} }
          
### TabixResultFiltDao:
   "TabixResultFiltDao": { 'const_arguments': [...], "columns" : {"pheno": "#pheno", "mlogp": "mlogp", "beta": "beta", "sebeta": "sebeta", "maf": "af_alt", "maf_cases": "af_alt_cases", "maf_controls": "af_alt_controls"}  }